wren-ui/src/apollo/server/utils/error.ts

```diff
@@ -8,6 +8,9 @@ export enum GeneralErrorCodes {
   NO_RELEVANT_DATA = 'NO_RELEVANT_DATA',
   NO_RELEVANT_SQL = 'NO_RELEVANT_SQL',
 
+  // Exception error for AI service (e.g., network connection error)
+  OTHERS = 'OTHERS',
```

Comment:
What does `OTHERS` means ? Can we give a more specific error ?

Reply:
I see. Didn't notice AI service error code called `OTHERS`. Maybe we can call it `AI_SERVICE_UNDEFINED_ERROR` in our enum ? since it basically means an error that AI service doesn't specifically define.

Reply:
Done by 7fb1273